cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js

```diff
@@ -65,12 +61,13 @@ describe("Feature - Event sign up : Tests execution date and time : " + new Date
 						eventSignup.getSignupForThisEventButton().click();
 						cy.VerifyEventName(eventName.text());
 						signedUpeventName = eventName.text().trim();
-						eventSignup.getFirstName().type(this.eventSignupTestData.firstName);
+						/*eventSignup.getFirstName().type(this.eventSignupTestData.firstName);
```

Comment:
Can we just remove this commented out code?

Reply:
Resolved